integration/pkg/api/oasis/genericoptions/values.go

```diff
@@ -0,0 +1,8 @@
+package genericoptions
+
+const (
```

Comment:
Please add comments here, otherwise `go-lint` will report warning.

Reply:
![image](https://user-images.githubusercontent.com/1905370/72229284-54d2b000-35e8-11ea-80f9-2e4c72937fb9.png)

Reply:
Add comments there.  Thanks for reminding.
Its strange I have the same config but I didn't see the warning, will try with different ways.